elements/pf-accordion/pf-accordion-header.css

```diff
@@ -53,7 +53,7 @@
   font-family:
     var(--pf-c-accordion__toggle--FontFamily,
       var(--pf-global--FontFamily--redhat-updated--heading--sans-serif,
-        "RedHatTextUpdated",
```

Comment:
this is actually incorrect?

https://www.patternfly.org/v4/components/accordion/ shows:
```css
font-family: "RedHatTextUpdated", "Overpass", overpass, helvetica, arial, sans-serif
```

Reply:
Taken from https://github.com/patternfly/patternfly-elements/issues/2397, we're not sending out a RedHatTextUpdated from the issue itself so we would need to update this to Red Hat Text.

Reply:
but accordion needs to match up with pf styles. better to load the updated font on our site rather than diverge, IMO

Reply:
I can see this both ways. In the past, we have said "if we can do better than the pfv4 components...", So I think you could definitely make the argument that this is one of those cases

On the other hand, the source material clearly specifies the "updated" font face. It seems hard to justify making this change just for our docs site, when we could just lost that font on the site. Pfe is supposed to be *visually* agnostic to upstream design choices so long as it doesn't affect accessibility

Reply:
@markcaron thoughts?  I agree with what Benny is saying here that pfe should be functionally the same as pf w/r/t styling but wanted to see your thoughts on updating it to RedHatText vs keeping it as RedHatTextUpdated?

Reply:
but why diverge from patternfly at all? that's why we have the RHDS font tokens

Reply:
> can we not add both to the font stack?

*we* can, but *PatternFly* doesn't, so we *shouldn't*. PFE is not supposed to have opinions on things like font stacks, and I don't want to maintain a "tweaks file" which specifies where and how PFE diverges from PatternFly design specs. If we want to change this in the font stack, we have to make a PR to PF core.